package.json

```diff
@@ -37,10 +37,11 @@
     "moment": "^2.29.4",
     "react": "^18.2.0",
     "react-big-calendar": "^1.5.2",
-    "react-datepicker": "^4.10.0",
+    "react-datepicker": "^4.11.0",
```

Comment:
Was this required?

Reply:
i think yarn did this, should I revert back?

Reply:
Looks like this is causing the test to fail, can you check please.

Reply:
yess